packages/core/test/controls/numericInputTests.tsx

```diff
@@ -0,0 +1,736 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import { expect } from "chai";
+import { mount, ReactWrapper } from "enzyme";
+import * as React from "react";
+
+import * as Errors from "../../src/common/errors";
+import { Classes, Keys, NumericInput, Position } from "../../src/index";
+
+describe("<NumericInput>", () => {
+
+    describe("Defaults", () => {
+
+        it("renders the buttons on the right by default", () => {
+            const component = mount(<NumericInput />);
+            const inputGroup = component.childAt(0);
+            expect(inputGroup.name()).to.equal("Blueprint.InputGroup");
+        });
+
+        it("has a stepSize of 1 by default", () => {
```

Comment:
Goal wasn't to test that `defaultProps` behave correctly; goal was to have tests in place that will yell loudly if someone happens to change default values that are explicitly documented in the component's props documentation (particularly values for `stepSize`, `minorStepSize` and `majorStepSize` in this case - if those were changed and there were no tests in place, downstream users could easily be affected without warning). I generally prefer that the whole surface area of a component be tested. But if you feel strongly, I can delete these tests.